WarpBuf/src/WarpBufChugin.h

```diff
@@ -64,29 +64,33 @@ class WarpBufChugin
     void setLoopEnd(double loopEnd) { m_clipInfo.loop_end = loopEnd; }
     bool getLoopEnable() { return m_clipInfo.loop_on; }
     void setLoopEnable(bool enable) { m_clipInfo.loop_on = enable; }
+
 private:
     // sample rate
     t_CKFLOAT m_srate;
 
     std::unique_ptr<RubberBand::RubberBandStretcher> m_rbstretcher;
 
+    // buffer related vars:
+    const int ibs = 1024;  // interleaved buffer size
```

Comment:
Imo ibs as a variable name was making it more difficult for me to read the code. What about something like interleaved_buffer_size or max_interleave_buffer since you already have a clarifying comment for what the acronym means